src/button/Button.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import FloatingActionButton from 'material-ui/FloatingActionButton';
+
+const Button = ({ raised, fab, color, disabled, onClick, children }) => {
+    let MuiButton;
+
+    if (fab) { // Always raised
+        MuiButton = FloatingActionButton;
+    } else if (raised) {
+        MuiButton = RaisedButton;
+    } else {
+        MuiButton = FlatButton;
+    }
+
+    const props = {
+        label: typeof children === 'string' ? children : null,
+        primary: color === 'primary' || null,
+        secondary: color === 'accent' || null,
+        disabled: disabled,
+        onTouchTap: onClick,
+    };
+
+    // Property gives error on FAB buttons in Material UI 0.19
+    if (fab) {
+        delete props.primary;
+    }
+
+    return (
+        <MuiButton {...props}>
+            {typeof children !== 'string' ? children : null}
+        </MuiButton>
+    );
+};
+
+Button.propTypes = {
+    /**
+     * If true, the button will use raised styling
+     */
+    raised: PropTypes.bool,
+
+    /**
+     * If true, the button will use floating action styling
+     */
+    fab: PropTypes.bool,
+
+    /**
+     *  If true, the button will be disabled
+     */
+    disabled: PropTypes.bool,
+
+    /**
+     * The theme color of the button ('default', 'primary', 'accent')
+     */
+    color: PropTypes.string,
```

Comment:
Here you could also use the `oneOf` prop type to check specifically for the valid values:

`PropTypes.oneOf(['default', 'primary', 'accent'])`